commons/phone/phone_contacts_list.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneContactsList",
```

Comment:
This should probably be called `PhoneContactList`. Also see the following comment.

Reply:
Renamed